modules/parquet/src/parquetjs/schema/types.ts

```diff
@@ -409,3 +424,32 @@ function fromPrimitive_INTERVAL(value: any) {
 
   return {months, days, milliseconds: millis};
 }
+
+export const PARQUET_TYPE_MAPPING: {[type in ParquetType]: typeof DataType} = {
```

Comment:
I feel this mapping should go into a separate file e.g. `lib/convert-schema.ts`.

The `parquetjs` directory contains our fork of the parquet code and we should not add our additional logic there.

Reply:
Moved to `lib/convert-schema.ts`